Week5/week-5.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f015db2f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Record Inserted successful\n",
+      "Enter Student ID:1\n",
+      "Enter Student Name: Om\n",
+      "Enter City Name: surat\n",
+      "Enter Contact Number:2435454657\n",
+      "Enter Student ID:2\n",
+      "Enter Student Name: sai\n",
+      "Enter City Name: Vyara\n",
+      "Enter Contact Number:123456778\n",
+      "Enter Student ID:3\n",
+      "Enter Student Name: Ram\n",
+      "Enter City Name: Bardoli\n",
+      "Enter Contact Number:5678901234\n",
+      "Enter Student ID:4\n",
+      "Enter Student Name: jai\n",
+      "Enter City Name: Navsari\n",
+      "Enter Contact Number:7890123456\n",
+      "Enter Student ID:5\n",
+      "Enter Student Name: Lalgopal\n",
+      "Enter City Name: Mathura\n",
+      "Enter Contact Number:9090898765\n",
+      "Record Inserted successful\n"
+     ]
+    }
+   ],
+   "source": [
+    "import csv\n",
+    "\n",
+    "with open(\"c:\\\\sqlite3\\\\student.csv\",\"w\",newline=\"\")as stud:\n",
+    "    write=csv.writer(stud)\n",
+    "    #5 record's insert directly in csv file\n",
+    "    header=['SId','SName','City','Contact']\n",
+    "    write.writerow(header)\n",
+    "    rows=[\n",
+    "          [1,'Om','Navsari',9086742578],\n",
+    "          [2,'Sai','Surat',9045316895],\n",
+    "          [3,'Ram','Bardoli',9997658431],\n",
+    "          [4,'Radha','Bardoli',673905315],\n",
+    "          [5,'Kishan','Surat',8904231578]\n",
+    "         ]\n",
+    "    write.writerows(rows)\n",
+    "    print('Record Inserted successful')\n",
+    "    #5 records insert using user input\n",
+    "    l=[]\n",
+    "    for record in range(5):\n",
+    "        n=int(input('Enter Student ID:'))\n",
+    "        name=input('Enter Student Name: ')\n",
+    "        city=input('Enter City Name: ')\n",
+    "        contact=int(input('Enter Contact Number:'))\n",
+    "        h=[n,name,city,contact]\n",
+    "        l.append(h)\n",
+    "    write.writerows(l)\n",
```

Comment:
whats difference between writerows() abd writerow()?

Reply:
Writerow() is used to write one row at a time. writerow is going to write a list of values into a single row
Writerows() is used to write many rows at once.writerows is going to write multiple rows from a buffer that contains one or more lists.